src/core/map/hooks/useMapEvents.ts

```
@@ -0,0 +1,52 @@
+import { useMapEffect } from './useMapEffect';
+import type { IMap } from '../providers/IMapProvider';
+
+export interface MapEventHandler {
+  event: string;
+  handler: (event: any) => boolean | void;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)